styles.css

```diff
@@ -50,35 +56,53 @@ nav ul li {
 }
 
 /* Button section */
-.button {
-  display: flex;
-  gap: 10px;
+.nav-button {
+  display: flex; /* Flexbox for aligning buttons in a row */
+  gap: 20px; /* Spacing between buttons */
+  align-items: center;
 }
 
-.button button {
-  background-color: var(--white);
-  border: none;
-  padding: 8px 12px;
-  border-radius: 5px;
-  font-size: 14px;
+.nav-button a, 
```

Comment:
Just one comment, not a blocker, but I would change the color of this link to be dark instead of blue 😉

Reply:
<img width="1440" alt="image" src="https://github.com/user-attachments/assets/e3eacb2c-e3b1-4bf2-bf04-a090e3a1f140">

Good job with the header @Rachealgit 🎉❤️